01/rmackley/wordvalue.py

```diff
@@ -0,0 +1,32 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words(dict):
+    """Load dictionary into a list and return list"""
+    dict_list = []
+    with open(dict) as f:
+        for line in f:
+            dict_list.append(line.rstrip())
+    return (dict_list)
+
+def calc_word_value(test_str):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    score = 0
+    for letter in test_str:
+        if letter not in [".",",","-","_",":",";"]:
```

Comment:
comment got deleted (?)
here you can use `punctuation` from the `string` module